velox/exec/Driver.cpp

```diff
@@ -194,7 +194,7 @@ void BlockingState::setResume(std::shared_ptr<BlockingState> state) {
         auto& driver = state->driver_;
         auto& task = driver->task();
 
-        std::lock_guard<std::mutex> l(task->mutex());
+        std::lock_guard<std::timed_mutex> l(task->mutex());
```

Comment:
@Yuhta we do see the problem in the production? Thanks!

Reply:
No it's for preventive measure, first to rule out the possibility that we get deadlock here during the detection (collecting run time of op call)